credentials.xml

```diff
@@ -0,0 +1,83 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<com.cloudbees.plugins.credentials.SystemCredentialsProvider plugin="credentials@2.1.14">
+  <domainCredentialsMap class="hudson.util.CopyOnWriteMap$Hash">
+    <entry>
+      <com.cloudbees.plugins.credentials.domains.Domain>
+        <specifications/>
+      </com.cloudbees.plugins.credentials.domains.Domain>
+      <java.util.concurrent.CopyOnWriteArrayList>
+        <com.cloudbees.jenkins.plugins.sshcredentials.impl.BasicSSHUserPrivateKey plugin="ssh-credentials@1.13">
+          <scope>GLOBAL</scope>
+          <id>6af1abaa-97d7-4ecf-9616-2d81faa76ac3</id>
+          <description>ubuntu_16_04_azure_slave</description>
+          <username>jenkins</username>
+          <privateKeySource class="com.cloudbees.jenkins.plugins.sshcredentials.impl.BasicSSHUserPrivateKey$UsersPrivateKeySource"/>
+        </com.cloudbees.jenkins.plugins.sshcredentials.impl.BasicSSHUserPrivateKey>
+        <org.jenkinsci.plugins.plaincredentials.impl.StringCredentialsImpl plugin="plain-credentials@1.4">
+          <scope>GLOBAL</scope>
+          <id>0fe2a31d-e947-47c3-aa08-6d5e451b61e5</id>
+          <description>jenkins-ci</description>
+          <secret>{AQAAABAAAAAw1wy6VRwbpPj52Ezeucpa8MEU+UjYHGjIYWKb5MvIc5kTeyj4EYLlhyP1ORTk50xD0GEJtRykMn0mpn5jzb+nhw==}</secret>
```

Comment:
Just carrying over a comment from the original PR https://github.com/clearcontainers/tests/pull/545
Just want to check we are sure we are not leaking any secrets - I believe we noted that the real secrets are stored in a separate secrets dir under Jenkins, and hence we are OK. Can we just confirm (as there are a lot of files here to check etc.).

Reply:
Yes, the secret that you can see here is encrypted with the keys stored inside the `secrets` folder, that's why it's okay to have the credentials.xml file here.

Reply:
thx - then lgtm - let me approve